### PR TITLE
Getting crushed by an airlock now has the possibility of removing a limb.

### PR DESCRIPTION
**Pull request by MrZoraman**
## About The Pull Request
Getting crushed by a door/firelock now has a 5% chance of amputating a random limb. There is a 0.01% chance of being beheaded by the door.
## Why It's Good For The Game
I think this makes the firelocks more realistic. I've often wondered "what does it mean to get crushed by the airlock? What does that actually look like?" And the picture that shows up in my mind is getting an arm or a leg caught in it. I kept the chance of losing a limb low because otherwise it would make space station 13 less fun, and I don't want that.
## Changelog
:cl:
add: Getting crushed by a firelock now has a small chance of amputating a random limb, and an even smaller chance of getting beheaded!
/:cl:

### CODE REVIEW

**Comment by nikothedude** [comment]
Bad idea, this is the ghetto treatment for dislocations, PLUS it already hardstuns for like, 8 seconds. Not really needed

File: code/game/machinery/doors/door.dm [hunk 1]
```diff
@@ -492,21 +498,65 @@
 		sleep(0.1 SECONDS)
 		open()
 
+/obj/machinery/door/proc/hurt_human(var/mob/living/carbon/victim)
+	if (prob(CRUSH_BEHEAD_CHANCE))
```
Comment (nikothedude) [inline code/game/machinery/doors/door.dm:502]: also for future reference its probably better to do a good 'ol

```
	if (prob(BASE_DELIMB_CHANCE))
	    if (head)
                  if (prob(BEHEAD_CHANCE))
```

so delimb chance isnt artificially reduced by someone having no head

**Comment by MrZoraman** [comment]
I see I made a mistake in my parameter declaration as well. I don't really want to make the whole ci/cd pipeline run again for that little change, since it sounds like this idea is pretty unpopular.

Thanks everyone for reviewing my PR!

**Comment by Lufferly** [comment]
This was a good idea and the percentages are so tame that it really wouldn't matter, doing ghetto surgery should be risky anyway. If you ever feel like opening this again I would also add some higher chances for some nasty wounds, but its up to you.

**Pull request closed without merging**